uimaj-core/src/main/java/org/apache/uima/analysis_engine/annotator/package.html

```diff
@@ -22,7 +22,7 @@
 <body>
 <p>The Annotator Interfaces, along with supporting interfaces
 and exception classes.</p>
-<p>The annotator interfaces are as follows:
+<p>The annotator interfaces are as follows:</p>
```

Comment:
Some sources (e.g. http://www.oracle.com/technetwork/articles/java/index-137868.html ) say to only use the <p> tag, not the </p>.  Also, I vaguely recall one of the javadoc lint checker tools complaining if you used some of these closing tags.  also this https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc says the doclet doesn't like the closing </p>

Reply:
To embed HTML in a comment, enclose it in single quotes (for inline HTML) or three single quotes to begin a verbatim block (and later again to end it.

Reply:
ok, my comment should say that the javadoc tool (at least for java 8, maybe for 9/10) complained if you put in the closing paragraph tag '</p>' .  So these probably should not be added?

Reply:
Hm... I guess "single quote" was not right and I should have said "backtick" (`)

The closing paragraph tag is not a problem. What JavaDoc 8+ is self-closing elements (`<br/>, <hr/>, etc.`).

Reply:
Mind that the closing tag was already there before, but it was in an illegal position **after** the `</ul>` - all I did is move it before the `<ul>`.

Reply:
ok, +1